.vuepress/public/postman/meilisearch-collection.json

```diff
@@ -374,20 +401,144 @@
 							],
 							"query": [
 								{
-									"key": "type",
+									"key": "uids",
+									"value": "3",
+									"disabled": true
+								},
+								{
+									"key": "indexUids",
+									"value": "books",
+									"disabled": true
+								},
+								{
+									"key": "types",
 									"value": "documentAdditionOrUpdate",
 									"disabled": true
 								},
 								{
-									"key": "status",
+									"key": "statuses",
 									"value": "failed",
 									"disabled": true
 								},
 								{
-									"key": "indexUid",
+									"key": "beforeEnqueuedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterEnqueuedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "beforeStartedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterStartedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "beforeFinishedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterFinishedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "canceledBy",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "limit",
+									"value": "2",
+									"disabled": true
+								},
+								{
+									"key": "from",
+									"value": "10",
+									"disabled": true
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Cancel tasks",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"url": {
+							"raw": "{{url}}/tasks",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"tasks"
+							],
+							"query": [
+								{
+									"key": "uids",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "indexUids",
 									"value": "books",
 									"disabled": true
 								},
+								{
+									"key": "types",
+									"value": "documentAdditionOrUpdate",
+									"disabled": true
+								},
+								{
+									"key": "statuses",
+									"value": "failed",
+									"disabled": true
+								},
+								{
+									"key": "beforeEnqueuedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterEnqueuedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "beforeStartedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterStartedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "beforeFinishedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterFinishedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "canceledBy",
```

Comment:
Can we use `canceledBy` with `/tasks/cancel?`?

Reply:
Yes, see [the spec](https://github.com/meilisearch/specifications/pull/195/files) and the [internal question](https://meilisearch.slack.com/archives/C044GEXNEUC/p1669112647316479) I asked following your question 😇

Reply:
Thanks!